cosmic/src/hrdiag.f

```diff
@@ -912,7 +913,8 @@ SUBROUTINE hrdiag(mass,aj,mt,tm,tn,tscls,lums,GB,zpars,
 * Use NS/BH masses given by Belczynski+08. PK.
 *
                      !First calculate the proto-core mass
-                     if(ecsn.gt.0.d0.and.mc.le.ecsn)then
+                     if(ecsn.gt.0.d0.and.mc.le.ecsn.and.
+     &                    mc.ge.ecsn_mlow)then
                         mcx = 1.38d0
                      elseif(ecsn.eq.0.d0.and.mc.le.2.25d0)then !this should be ecsn, unless ecsn=0
```

Comment:
We pulled lines 919-920 out in an earlier code block -- is there a reason that this block keeps them in?

Reply:
@katiebreivik I forgot to take these out for the Belczynski prescription remnantflag=2 (both in the normal and stripped-star cases). I've removed these in the most recent commit.